### PR TITLE
Add msg with receipt link for sharing

### LINKED ISSUE #1238: Add additional text with the recipt link when sharing
**Summary:**

The receipt link that can be shared needs some extra message text rather than simple link while sharing.

**Steps to reproduce:**

1. Go to receipt activity,
2. share the receipt link given and observe.

**Expected behavior:**

There should be an extra message with the link it could be a promotional message or a simple sender message.

**Observed behavior:**

Right now there is no extra message with the link

**Device and Android version:**

VIVO Y83, Android Version = 8.1.1

**Screenshots:**

### PR DESCRIPTION
**Pull request by EGOR-IND**
## Issue Fix
Fixes #1238 

## Description
Added the message with receipt link for sharing purposes.

##

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by EGOR-IND** [comment]
Hey @devansh-299 I made the PR for the message with receipt link, let me know if you need any changes:).

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/receipt/ui/ReceiptActivity.java [hunk 1]
```diff
@@ -199,9 +199,15 @@ void copyReceiptLink() {
 
     @OnClick(R.id.transaction_reciept_share)
     void shareReceiptLink() {
+        String senderMsg =
+                "Here's the receipt link for the transaction made using MifosPay from payer "
```
Comment (s-ayush2903) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/receipt/ui/ReceiptActivity.java]: Avoid hardcoding as usual : )

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/receipt/ui/ReceiptActivity.java [hunk 2]
```diff
@@ -199,9 +199,15 @@ void copyReceiptLink() {
 
     @OnClick(R.id.transaction_reciept_share)
     void shareReceiptLink() {
+        String senderMsg =
```
Comment (s-ayush2903) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/receipt/ui/ReceiptActivity.java]: Also the variable name _senderMsg_ doesn't seem to make much sense(at least to me), as this was never written by the sender. As this is the content/body of the mail, so it should be named according to that in my opinion

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/receipt/ui/ReceiptActivity.java (hunk 1 above)
Comment (EGOR-IND) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/receipt/ui/ReceiptActivity.java]: okay, I'll make a string constant for that thanks for reminding i totally forgot:).

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/receipt/ui/ReceiptActivity.java (hunk 2 above)
Comment (EGOR-IND) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/receipt/ui/ReceiptActivity.java]: what about shareMsgBody?

**Review by Supriyo9 — APPROVED** [review]

**Comment by Prashant830** [comment]
@devansh-299  please close this pr this issue solved.

**Comment by devansh-299** [comment]
Closing this pull request, refer this [link](https://github.com/openMF/mobile-wallet/pull/1239#issuecomment-1207411381)

**Pull request closed without merging**